src/store/sidebarStore.js

```diff
@@ -21,13 +21,17 @@
  */
 
 const state = {
-	show: true,
+	show: false,
```

Comment:
I strongly vote against this. In a call maybe, but showing even less of the information and more white space... why?

Reply:
Yep, I'd say it's the same argumentation than the appnavigation. Doesn't make sense to enforce fullscreen if there is enough space to display everything.

Reply:
For the sake of progressive disclosure. The fact that there's enough space isn't enough of a reason to show secondary information.
For those who want it open all the time, no problem, they just have to open it once and then we're going to remember that!

Reply:
You loose the information who is currently active in the conversation and reading the chat. That's quite a huge drawback to me and the gain is just more whitespace.
I get it when you are on <1280px or something, but I'm on 4k or 1/2 of 4k and dont want to have to open it once everytime.

Reply:
You just need to open it once, not everytime

Reply:
> Let's say you have an 8k screen, would you be happy to see photoshop displaying every single panel just because there's space for it?

Not a proper comparison because panels are tabbed in Photoshop, same as we do. So yes, I'd be happy to have my left panel shown by default.



> The fact that there's enough space isn't enough of a reason to show secondary information by default.
> L

I can tell you that I will rant every time I have to manually open it because in call that is everything I want to look at. Remember that this is not "secondary informations", there is the chat here that is also used during calls.

Reply:
> Not a proper comparison because panels are tabbed in Photoshop, same as we do. So yes, I'd be happy to have my right panel shown by default.

In PS you have panels containing tabs containing more panels. I wasn't trying to draw the perfect analogy btw, just trying to make you see the value of presenting a less complex layout by default. The gain is not merely whitespace, it's simplicity and a more streamlined experience.

>I will rant every time I have to manually open

Why do you both keep saying "everytime"? I said "just once" and then we keep your preference stored in the browser!

Reply:
> Why do you both keep saying "everytime"? I said "just once" and then we keep your preference stored in the browser!

Because that was not specified in the first place?
Anyway, I gave my opinion, I'm out. No time for arguing ^^

Reply:
> For those who want it open all the time, no problem, they just have to open it once and then we're going to remember that!

> You just need to open it once, not everytime

Reply:
> The fact that there's enough space isn't enough of a reason to show secondary information by default.

Strongly agree here with @ma12-co. Check Slack, or Signal, or WhatsApp, none of them show any user list by default as it’s advanced info.